cmd/jaeger/collector-with-kafka.yaml

```diff
@@ -0,0 +1,28 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+
+processors:
+  batch:
+
+exporters:
+  kafka:
+    brokers:
+      - localhost:9092
+    topic: "jaeger-spans"
+    encoding: otlp_proto
+
+service:
```

Comment:
please keep overall config layout consistent with other config files. Specifically, I prefer service: section to be at the top, as it provides the overview of the whole setup.

Reply:
okay, noted!